apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -323,23 +323,42 @@ public override Task<int> ExecuteNonQueryAsync(CancellationToken cancellationTok
             if (spannerCommandTextBuilder.IsCreateDatabaseCommand)
             {
                 var parent = new InstanceName(spannerConnection.Project, spannerConnection.SpannerInstance);
-                var response = await databaseAdminClient.CreateDatabaseAsync(parent, commandText).ConfigureAwait(false);
+                var response = await databaseAdminClient.CreateDatabaseAsync(
```

Comment:
Can the multiple commands create multiple databases? What about "create table" (for a connection with a database) followed by "create database" (creating a separate one)? Hopefully we don't need to worry about these and the server will just Do The Right Thing.

Reply:
hmm create database cannot be one of the extra ddl statements (it must be the first one if specified).  I will make that clear in the docs.